Revert "build,ui: Require & link to gtk+-3.0"
(commit b75d2eb469414fd677e3962c879b170c6bffe7d7)

diff --git b/configure.ac a/configure.ac
index 27a76ab..f56fe96 100644
--- b/configure.ac
+++ a/configure.ac
@@ -30,7 +30,7 @@ GIO_REQUIRED=2.16
 GEE_REQUIRED=0.5.0
 UUID_REQUIRED=1.41.3
 LIBSOUP_REQUIRED=2.26.0
-GTK_REQUIRED=2.90.3
+GTK_REQUIRED=2.16
 LIBSQLITE3_REQUIRED=3.5
 
 PKG_CHECK_MODULES(LIBGUPNP, gupnp-1.0 >= $GUPNP_REQUIRED)
@@ -159,18 +159,18 @@ AC_ARG_WITH( ui,
 HAVE_GTK=no
 if test x$try_ui = xyes ; then
     dnl check for gtk
-    PKG_CHECK_MODULES(GTK, gtk+-3.0 >= $GTK_REQUIRED ,
+    PKG_CHECK_MODULES(GTK, gtk+-2.0 >= $GTK_REQUIRED ,
       [
         AC_SUBST(GTK_CFLAGS)
         AC_SUBST(GTK_LIBS)
         if test x$enable_vala = xyes ; then
-                AC_CHECK_FILE("${VAPIDIR}/gtk+-3.0.vapi",
+                AC_CHECK_FILE("${VAPIDIR}/gtk+-2.0.vapi",
                   [
                         HAVE_GTK=yes
                   ],
                   [
                         HAVE_GTK=no
-                        AC_MSG_ERROR("Unable to find Vala bindings for gtk+-3.0")
+                        AC_MSG_ERROR("Unable to find Vala bindings for gtk+-2.0")
                   ])
         else
                 HAVE_GTK=yes
diff --git b/src/ui/Makefile.am a/src/ui/Makefile.am
index 3ca3f3b..946639b 100644
--- b/src/ui/Makefile.am
+++ a/src/ui/Makefile.am
@@ -35,7 +35,7 @@ rygel_preferences_VALAFLAGS = \
 	--vapidir=$(rygeldir) --thread \
 	--pkg rygel-1.0 --pkg rygel-build-config --pkg gupnp-1.0 \
 	--pkg gupnp-av-1.0 --pkg dbus-glib-1 --pkg gconf-2.0 \
-	--pkg gstreamer-0.10 --pkg gio-2.0 --pkg gee-1.0 --pkg gtk+-3.0 -g
+	--pkg gstreamer-0.10 --pkg gio-2.0 --pkg gee-1.0 --pkg gtk+-2.0 -g
 
 rygel_preferences_LDADD = \
 		$(abs_top_builddir)/src/rygel/librygel-configuration.a \
